pandas/core/common.py

```diff
@@ -97,6 +97,13 @@ class UnsupportedFunctionCall(ValueError):
     pass
 
 
+class UnsortedIndexError(KeyError):
+    """ This error is raised when attempting to get a slice of a MultiIndex
+    and the index has not been lexsorted.
+    """
```

Comment:
"This error is raised when" -> "Error raised when"

Can you also add that it is a subclass of a KeyError?